src/components/InfoViewer/formatters/pqGroup.ts

```diff
@@ -42,7 +42,7 @@ export const formatPQTabletConfig = createInfoFormatter<TPQTabletConfig>({
 export const formatPQPartitionConfig = createInfoFormatter<TPQPartitionConfig>({
     values: {
         StorageLimitBytes: formatBytes,
-        WriteSpeedInBytesPerSecond: formatBps,
+        WriteSpeedInBytesPerSecond: (value) => value && formatBps(value),
```

Comment:
maybe it is possible to check if not value in `formatBps`?

Reply:
Yep, added check there